src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -784,11 +772,8 @@ namespace ExhaustAirSystemManager {
         thisExhOutlet.Quality = thisExhInlet.Quality;
         thisExhOutlet.Press = thisExhInlet.Press;
 
-        // More node elements
-        thisExhOutlet.MassFlowRateMax = thisExhInlet.MassFlowRateMax;
-        thisExhOutlet.MassFlowRateMaxAvail = thisExhInlet.MassFlowRateMaxAvail;
-
         // Set the Node Flow Control Variables from the Fan Control Variables
+        thisExhOutlet.MassFlowRateMax = thisExhInlet.MassFlowRateMax;
         thisExhOutlet.MassFlowRateMaxAvail = thisExhInlet.MassFlowRateMaxAvail;
         thisExhOutlet.MassFlowRateMinAvail = thisExhInlet.MassFlowRateMinAvail;
```

Comment:
[src/EnergyPlus/ExhaustAirSystemManager.cc:792]:(style),[redundantAssignment],Variable 'thisExhOutlet.MassFlowRateMaxAvail' is reassigned a value before the old one has been used.